src/components/create-filter-form.js

```diff
@@ -164,13 +220,36 @@ const CreateFilterDialog = ({
                 <Grid container direction="row" spacing={1}>
                     <Grid item xs={12} sm={8}>
                         <TextField
-                            style={{ width: '100%' }}
+                            style={{ width: '90%' }}
                             defaultValue={''}
                             onChange={(event) =>
-                                onChangeInputName(event.target.value)
+                                handleFilterNameChanges(event.target.value)
                             }
                             label={inputLabelText}
                         />
+                        {loadingCheckFilterName && (
```

Comment:
Maybe move this in a `renderFilterNameStatus` then you could be sure that there will be only one div with CircularProgress if loadingCheckFilterName else with CheckIcon if filterNameChecked or even a red cross if not.